src/platform/ios/ios.m

```diff
@@ -0,0 +1,75 @@
+#import "ios.h"
+#import "SDL.h"
+#import "SDL_syswm.h"
+#import "SDL_video.h"
+#import "SDL_system.h"
+#import "SDL_messagebox.h"
+#import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
+
+#import "GameDataPickerController.h"
+
+#define GAME_PATH_MAX 300
+static int has_directory;
+static char path[GAME_PATH_MAX];
+
+void c3_path_chosen(char *new_path) {
+    unsigned long len = strlen(new_path);
+    strncpy(path, new_path, len);
```

Comment:
I don't think I've ever done this in Julius, but only in Augustus, but still, `snprintf` Is safer.

Also, if `len` is larger than `GAME_PATH_MAX`, you got an issue.

```suggestion
    snprintf(path, GAME_PATH_MAX, "%s", new_path);
```

Reply:
good point, thanks!